src/db/spanner/models.rs

```diff
@@ -647,12 +650,21 @@ impl SpannerDb {
         }
     }
 
+    fn erect_tombstone(&self, user_id: u32) -> Result<()> {
+        self.sql("INSERT INTO user_collections (userid, collection, last_modified) values (@userid, \"0\", @modified)")?
```

Comment:
Should be
```suggestion
        self.sql("INSERT INTO user_collections (userid, collection, last_modified) values (@userid, 0, @modified)")?
```
but might as well bind the TOMBSTONE const instead since it's defined (surprised clippy didn't complain about it being unused)